CHANGES_UNRELEASED.md

```diff
@@ -17,3 +17,4 @@ Please call `Viaduct.shared.useReqwestBackend()` as soon as possible before usin
 
 - A new function was added to return a list of duplicate logins, ignoring
   username. ([#2542](https://github.com/mozilla/application-services/pull/2542))
+- Added an optional `ttl` parameter to `getAccessToken`, to limit the cache length of the token.
```

Comment:
This entry belongs in a different category (FxA Client), see `CHANGELOG.md` for examples.

Reply:
:+1: